testing/test_modules.out

```diff
@@ -5,12 +5,19 @@ RESULT skip base.startup.dhcp Using static ip 10.20.0.5
 RESULT skip base.switch.ping No local IP has been set, check ext_loip in system.conf
 RESULT pass base.target.ping target reached %% 10.20.0.5
 Testing tls
-RESULT skip security.tls.v3
-RESULT skip security.x509
+Module execution failed.
```

Comment:
why is the module failing?  That doesn't seem right!

Reply:
What determines a module to pass or fail?  This seems to happen outside of the tls test so I can't really comment on what is determining this.